CMakeLists.txt

```diff
@@ -361,7 +361,9 @@ foreach (s ${COMPILETIME_OPTIONS_LIST})
 	set (COMPILETIME_OPTIONS "${COMPILETIME_OPTIONS} -D${s}")
 endforeach (s)
 
-generate_and_install_pkg_config_file(torrent-rasterbar libtorrent-rasterbar)
+if (NOT MSVC)
```

Comment:
>Or are pkg-config files used in MSYS?

AFAIK yes, they are more like linux on this matter.

>If so, is MSVC incompatible with the MSYS environment? 

IIRC I've once saw someone using cl.exe within msys env, but personally I consider that an exotic combination... 

Another point is msvc compiler flags are different than gcc/clang and even if the pkg-config is generated, it won't be compatible to other compilers. Also I suppose the majority who use msvc won't bother about pkg-config.

Reply:
I should also mention that pkg-config generation doesn't even work when compiling with MSVC, failing the build with some error message, based my own experience I choose to avoid the generation instead of trying fixing it.

Reply:
alright. @zeule is msvc expected to support pkg-config? if not, it seems reasonable to exclude it. I still think it would be good to have a comment though, to explain why it's excluded

Reply:
As @Chocobo1 has pointed out above, if GCC comand line options do not wok with MSVC, generating it makes no sense.